### PR TITLE
Added a new video

### PR DESCRIPTION
**Pull request by arjunpandey2003**
ISSUE #324

### CODE REVIEW

**Review by 0x48piraj — CHANGES_REQUESTED** [review]
**Travis Tests** are failing!

**Review by the-codingninja — CHANGES_REQUESTED** [review]
Please correct video.yml to fix travis

File: _data/video.yml [hunk 1]
```diff
@@ -3,3 +3,8 @@
   title: About FOSSASIA
   img: FOSSASIA.jpeg
   author_name: Supun Tharinda Edirisuriya
+  
+- url: https://www.youtube.com/watch?v=Noy7OnKIhXM&t=19s
```
Comment (anshumanv) [inline _data/video.yml:7]: Correct link - https://www.youtube.com/watch?v=Noy7OnKIhXM
The link you gave transitions to t=19 seconds.

**Review by anshumanv — CHANGES_REQUESTED** [review]
Check comments

**Comment by ParthS007** [comment]
closing this as a duplicate of #405

**Pull request closed without merging**